goth/payment_config.py

```diff
@@ -18,19 +18,21 @@
     },
     "erc20": {
         "env": {
-            "YA_PAYMENT_NETWORK": "mainnet",
+            "ERC20_MAINNET_REQUIRED_CONFIRMATIONS": 0,
```

Comment:
I think our future selves might love something like:
```
# Some_comment_why_we_set_this_to_0
# (Mostly because this variable is confusing, I think?)
REQUIRED_CONFIRMATIONS = 0

...

    "ERC20_MAINNET_REQUIRED_CONFIRMATIONS": REQUIRED_CONFIRMATIONS,
    
...

    "ERC20_POLYGON_REQUIRED_CONFIRMATIONS": REQUIRED_CONFIRMATIONS,
    
```

Reply:
c6a035f